src/services/reallocateCourtCaseToForce.ts

```diff
@@ -60,7 +61,13 @@ const reallocateCourtCaseToForce = async (
         }
       }
 
-      const unlockResult = await unlockCourtCase(entityManager, +courtCaseId, user)
+      const unlockResult = await updateLockStatusToUnlocked(
+        entityManager,
+        +courtCaseId,
+        user,
+        UnlockReason.TriggerAndException,
+        [] //TODO pass an actual audit log events array
```

Comment:
Does this need to be completed?

Reply:
We decided to do a PR just for unlocking because it involved quite a bit of a refactor, more PR coming soon!